app/scripts/modules/core/src/utils/debug.ts

```diff
@@ -22,7 +22,7 @@ const getMethodName = (target: any, propertyKey: string) => {
 export const DebugTiming = (label?: string) => (target: any, propertyKey: string, descriptor: PropertyDescriptor) => {
   const fn = descriptor.value;
   label = padStart(label ? label : getMethodName(target, propertyKey), 50);
-  descriptor.value = function DebugTiming() {
+  descriptor.value = function() {
```

Comment:
guessing these were named functions to aid in performance analysis (@christopherthielen would know) - if it's no longer relevant, might as well make them arrow functions